src/facade/files/__name@dasherize__/__name@dasherize__.facade.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Store } from '@ngrx/store';<% if (store) { %>
+import { <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageSelectors } from './shared/store';<% } %><% if (isNgrxInstalled) { %>
+import { AppState } from '@shared/store';<% } %>
+
+@Injectable()
+export class <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageFacade {
+  <% if (store) { %>public get isLoading$(): Observable<boolean> {
+    return this.store.select(<%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageSelectors.isLoading);
```

Comment:
@Goodmain same as above, this `classify` call can be shortened.

Reply:
@dmitryusenko Fixed